tests/test_parametric_workflow.py

```diff
@@ -119,7 +119,7 @@
 study_1 = ParametricStudy(session.solver.root.parametric_studies).initialize()
 
 parametricStudies_exp = 1
-parametricStudies_test = len(ParametricStudy._all_studies.keys())
+parametricStudies_test = len(study_1.get_all_studies().keys())
```

Comment:
I will remove these changes from the PR.. I have let @akulshre-qa know what is required in his PR

Reply:
Thanks @seanpearsonuk I will update my PR.